Packs/Devo/Integrations/Devo_v2/Devo_v2.yml

```diff
@@ -4,193 +4,212 @@ commonfields:
+          description: The link to the Devo table for executed query.
+      description: Queries Devo based on the specified LINQ query.
+    - name: devo-get-alerts
+      arguments:
+        - name: table_name
+          description: name of alert table to fetch alerts from a table. If not provided 'siem.logtrust.alert.info' will be used.
+        - name: prefix
+          description: This is required when user wants to fetch from custom alert tables.
+        - name: from
+          description: Start datetime for alerts to fetch.
+          required: true
+        - name: to
+          description: End datetime for alerts to fetch.
+        - name: items_per_page
+          description: Enter the per page value you want to set.
+          required: true
+        - name: filters
+          description: Key value filter to apply to retrieve the specified alerts. For more information, see the Devo documentation.
+          isArray: true
+        - name: queryTimeout
+          description: Timeout in seconds for this query to run against Devo to override the minute default in the platform.
+          defaultValue: "60"
+        - name: writeToContext
+          description: Whether to write results to context. Can be "true" or "false". The default value is "true".
+          defaultValue: "true"
+        - name: linqLinkBase
+          description: Overrides the global Devo base domain for linq linking.
+      outputs:
+        - contextPath: Devo.AlertsResults
+          description: List of dictionary alerts from the specified time range.
+        - contextPath: Devo.QueryLink
+          description: The link to the Devo table for the executed query.
+      description: Queries alerts in the specified timeframe.
+    - name: devo-multi-table-query
+      arguments:
+        - name: tables
+          required: true
+          description: A list of table names to check for the searchToken.
+          isArray: true
+        - name: searchToken
+          required: true
+          description: A string to search for in the specified tables (in any column).
+        - name: from
+          required: true
+          description: Start datetime for the specified query. This argument supports natural language (e.g., 2 day, 3 week), Unix timestamps, Python datetime objects, and string datetimes
+        - name: to
+          description: End datetime for specified query. If provided must be in same format as "from" argument. This argument is ignored in a date range.
+        - name: queryTimeout
+          description: Timeout in seconds for this query to run against Devo to override the minute default in the platform.
+          defaultValue: "60"
+        - name: writeToContext
+          description: Whether to write results to context. Can be "true" or "false". The default value is "true".
+          defaultValue: "true"
+        - name: items_per_page
+          description: Enter the per page value you want to set.
+          required: true
```

Comment:
same

Reply:
Also, Is it instead of the limit argument? If so, then what customers with the limit argument should do? their command will fail.

Reply:
Fixed in commit acb5c0dc46d45695be9bf68f589b11e255e3d3e5